app/controllers/servers_controller.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+class ServersController < ApplicationController
+  include ApiHelper
+
+  before_action :set_server, only: [:update, :destroy, :panic]
+
+  # Return a list of the configured BigBlueButton servers
+  # GET /bigbluebutton/api/v1/servers
+  #
+  # Successful response:
+  # [
+  #   {
+  #     "id": String,
+  #     "url": String,
+  #     "secret": String,
+  #     "state": String,
+  #     "load": String,
+  #     "load_multiplier": String,
+  #     "online": String
+  #   },
+  #   ...
+  # ]
+  def index
+    servers = Server.all
+
+    if servers.empty?
+      render json: { message: 'No servers are configured' }, status: :ok
+    else
+      server_list = servers.map do |server|
+        {
+          id: server.id,
+          url: server.url,
+          secret: server.secret,
+          state: if server.state.present?
+                   server.state
+                 else
+                   server.enabled ? 'enabled' : 'disabled'
+                 end,
```

Comment:
Can this be cleaned up with `server.state.presence ||  server.enabled ? 'enabled' : 'disabled'`?